tools/manage_modules.php

```diff
@@ -57,6 +57,13 @@
             if ($dryrun) {
                 continue;
             }
+            $ModuleID = $DB->pselectOne(
+                "SELECT ID FROM modules WHERE Name=:name",
+                ['name'=>$module]
+            );
+
+            $DB->delete("issues", [ 'module' => $ModuleID]);
+            $DB->delete("permissions", [ 'moduleID' => $ModuleID]);
```

Comment:
I don't know if this is a good idea or scaleable because it would need to be updated for every module (and it's not undoable). It also means if you disable and then reenable a module you lose all the issues and permissions.

The linked issue says to just put a try/catch around the delete and log an error if it can't delete it.

@CamilleBeau do you have any thoughts?

Reply:
@driusan Yes I second that